.test-infra/dataproc/init-actions/flink.sh

```diff
@@ -155,7 +155,8 @@ function configure_flink() {
 # Settings applied by Cloud Dataproc initialization action
 jobmanager.rpc.address: ${master_hostname}
 jobmanager.heap.mb: ${flink_jobmanager_memory}
-taskmanager.heap.mb: ${flink_taskmanager_memory}
+taskmanager.memory.process.mb: ${flink_taskmanager_memory}
+taskmanager.memory.jvm-metaspace.mb: 512
```

Comment:
The correct option name is: `taskmanager.memory.jvm-metaspace.size`: https://ci.apache.org/projects/flink/flink-docs-release-1.10/ops/config.html#taskmanager-memory-jvm-metaspace-size